ui/revenuecatui/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="restore_purchases">Restore purchases</string>
```

Comment:
I've only added the default (english) translation right now, but will need to add other translations to test localization properly for hardcoded strings.

Reply:
👍🏻